README.md

```diff
@@ -149,33 +246,39 @@ See the [Springroll Application Docs](https://github.com/SpringRoll/SpringRoll/t
 
 
 ### UISizePlugin, LayersPlugin:
+The UI size plugin allows users to control the size of custom pointers and buttons within the game. The size value ranges from 0 to 1, defaulting to 0.5.
+
+The Layers plugin allows users to hide distracting layers within a game. This is implemented as a slider as well, from 0 to 1. 0 indicates "show all layers"
```

Comment:
Probably don't need the 'This is implemented as a slider as well' since line 254 says that they expect range inputs and the UI size plugin doesn't mention a range input.